packages/models-library/tests/test_api_schemas_catalog.py

```diff
@@ -21,7 +21,7 @@ def test_service_port_with_file():
         }
     )
 
-    port = ServicePortGet.from_service_io("input", "input_1", io).model_dump(
+    port = ServicePortGet.from_domain_model("input", "input_1", io).model_dump(
```

Comment:
it becomes unclear... and a bit contra-intuitive as you usually add `Str` to types. now for the functions you go for something where I need to hover to check the type. just a thought, but that is a bit inconsistent. No need to change, or so, that is just a thought. It is ok for me.

Reply:
- I add `Str` suffix to string-like types or when I create aliases for clarity. This suffix is borrowed from pydantic's naming e.g. https://docs.pydantic.dev/2.0/usage/types/string_types/
- In this case the first argument is a Literal. I will change it to make it more clear.